config/config.go

```diff
@@ -821,6 +821,7 @@ type MempoolConfig struct {
 	// Do not remove invalid transactions from the cache (default: false)
 	// Set to true if it's not possible for any invalid transaction to become
 	// valid again in the future.
+	// Deprecated: KeepInvalidTxsInCache is deprecated and will be removed in a future version.
```

Comment:
can we also deprecate it in the `toml.go` where we keep the configs?

Reply:
done https://github.com/celestiaorg/celestia-core/pull/2623/commits/d9926b534652f27ddf6817e686e445f338461f32